### DISCUSSION TITLE
0.2.1

### DISCUSSION THREAD

**Discussion by zecruel** [zecruel/CadZinho#31] [Announcements]
Some bug fixes release.

For issues related to the standard OpenGL driver on Windows, I recommend using the Mesa3D driver, available at [Mesa3D](https://github.com/pal1000/mesa-dist-win/releases/latest) (download mesa3d-XX.XX.XX-release-mingw.7z then extract DLL files to main CadZinho folder)

<hr /><em>This discussion was created from the release <a href='https://github.com/zecruel/CadZinho/releases/tag/0.2.1'>0.2.1</a>.</em>

**Comment by zvezdochiot** [comment]
Hi @zecruel .

Release 0.2.1.  Windows x86_64:  
Depends: KERNEL32.dll, SHELL32.dll, msvcrt.dll, OPENGL32.dll, glew32.dll, SDL2.dll.

**OPENGL32.dll** should not have been archived?

**Comment by zecruel** [comment]
Hi @zvezdochiot .

OPENGL32.dll is a standard Windows system file, and is specific to the machine's video card driver. But you can use the Mesa3D driver if you have a problem with the default and don't want to update it.
